platform/android/build.gradle.template

```diff
@@ -35,32 +35,38 @@ android {
 	packagingOptions {
 		exclude 'META-INF/LICENSE'
 		exclude 'META-INF/NOTICE'
+		exclude 'res/drawable/icon.png'
+		exclude 'res/layout/downloading_expansion.xml'
+		exclude 'res/layout/status_bar_ongoing_event_progress_bar.xml'
 	}
+
 	defaultConfig {
-		minSdkVersion 14
+		minSdkVersion 10
```

Comment:
Give Android 2.3.3 Users a second chance to enjoy Godot powered Apps/Games. This doesn't mean Godot MUST/SHOULD support API-10. AFAIU Godot runs quite well on Gingerbread. By showing All Godot capabilities, I hope it could attract even more Android developers to use Godot.

This could be changed eventually when Godot can't be played on Android API-10 powered devices.

:-)